Makefile

```diff
@@ -55,11 +55,11 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+test check:
```

Comment:
oh, i added this according to the comments 
```
#   check: Run tests.
#   test: Run tests.
```

agree, fixed. origin also did like what your specified. thanks.